tests/codegen/issues/issue-109328-split_first.rs

```diff
@@ -0,0 +1,16 @@
+//@ compile-flags: -O
+//@ min-llvm-version: 17
```

Comment:
Remove the `min-llvm-version: 17` lines, as this is the minimum supported version anyway. (Here and in some tests below as well.)